nise/generators/azure/__init__.py

```diff
@@ -15,8 +15,8 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """Module for azure data generators."""
-from nise.generators.azure.azure_generator import AZURE_COLUMNS  # noqa: F401
-from nise.generators.azure.azure_generator import AZURE_COLUMNS_V2  # noqa: F401
+from nise.generators.azure.azure_generator import AZURE_COLUMNS_V2_RESOURCE_GROUP  # noqa: F401
+from nise.generators.azure.azure_generator import AZURE_COLUMNS_V2_SUBSCRIPTION  # noqa: F401
```

Comment:
We could add a per-file-ignore for F401 on this file (or all `__init__.py` files) and remove the inline ignores.
```patch
diff --git tox.ini tox.ini
index 2b1e4c6..1651689 100644
--- tox.ini
+++ tox.ini
@@ -11,6 +11,8 @@ max-complexity = 10
 max-line-length = 120
 import-order-style = pycharm
 application-import-names = nise
+per-file-ignores =
+    __init__.py: F401
 
 [testenv]
 passenv = CI TRAVIS TRAVIS_*

```

Reply:
Suggestion accepted, ty